### PR TITLE
updated missing info about on-premises

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1966

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Cloud only and hybrid deployments provide many choices for multi-factor authenti
 
 #### Directory synchronization
 
-Hybrid and on-premises deployments use directory synchronization, however, each for a different purpose.  Hybrid deployments use Azure Active Directory Connect to synchronize Active Directory identities or credentials between itself and Azure Active Directory. This helps enable single sign-on to Azure Active Directory and its federated components.
+Hybrid and on-premises deployments use directory synchronization, however, each for a different purpose.  Hybrid deployments use Azure Active Directory Connect to synchronize Active Directory identities or credentials between itself and Azure Active Directory. This helps enable single sign-on to Azure Active Directory and its federated components. On-premises deployments use directory synchronization to import users from Active Directory to the Azure MFA Server, which sends data to the Azure MFA cloud service to perform the verification.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]: Copy checked - no changes.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #1966?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #1966?

Thank you.

**Comment by DaniHalfin** [comment]
Looks about right. @mapalko - please confirm. Thanks :)

**Comment by mapalko** [comment]
@DaniHalfin looks good

**Pull request merged**